rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
```

Comment:
- [ ] The initialize method of Rental should be modified so `Book` and `Person` are set in it to break the many-to-many relationship. Link to the example article: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`. See suggestion below:

```suggestion
  def initialize(date, person, book)
    @date = date
    @person = person
    person.rentals << self
    @book = book
    book.rentals << self
  end
```